public/app/features/commandPalette/actions/useActions.ts

```diff
@@ -0,0 +1,49 @@
+import debounce from 'debounce-promise';
+import { useEffect, useMemo, useState } from 'react';
+
+import { useSelector } from 'app/types';
+
+import { CommandPaletteAction } from '../types';
+
+import { getDashboardSearchResultActions, getRecentDashboardActions } from './dashboardActions';
+import getStaticActions from './staticActions';
+
+const debouncedDashboardSearch = debounce(getDashboardSearchResultActions, 100);
```

Comment:
think we should be a little more aggressive with our debouncing here btw, 100 is very short. 200 seems like a better compromise between frontend/backend perf? 🤔